test/unit/test_put_get.py

```diff
@@ -70,6 +70,37 @@ def test_put_error(tmpdir):
     chmod(file1, 0o700)
 
 
+def test_get_empty_file(tmpdir):
+    """Tests for error message when retrieving missing file."""
+    tmp_dir = str(tmpdir.mkdir("getfiledir"))
+
+    con = MagicMock()
+    cursor = con.cursor()
+    cursor.errorhandler = Error.default_errorhandler
+    query = "PUT something"
```

Comment:
Not super important, but let's just to be less confusing
```suggestion
    query = f"GET something file:\\{tmp_dir}"
```